### PR TITLE
Feature/speed-ops

### PR DESCRIPTION
**Pull request by heet-git**
Hello team.

I created speed slider and Operation drop-down for the game. Please, review the PR and let me know if there is any mistake or change needed.

### CODE REVIEW

File: boolebots/src/components/Speed.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+ useEffect(() => {
+    const interval = setInterval(() => {
+      // Move bots here with the current speed value
+    }, 1000 / speed); // Adjust bot movement speed based on the selected speed
+
+    return () => clearInterval(interval);
+  }, [speed]);
+
+    return (<input
```
Comment (demartinom) [inline boolebots/src/components/Speed.js:15]: I think maybe some kind of indication of speed would be helpful? Like have below it say slow med fast.

Comment (demartinom) [reply boolebots/src/components/Speed.js:15]: Even just something like this (but it should probably say what they'r
Screen Shot 2023-05-15 at 2 59 23 PM
e in seconds)

Comment (heet-git) [reply boolebots/src/components/Speed.js:15]: Hey Matt. Thanks for the review. I have added this speed indicator and also I forgot to render the operation drop-down.

**Review by demartinom — APPROVED** [review]
Looks great Heet!

**Pull request merged**